app/controllers/media_controller.rb

```diff
@@ -63,6 +63,9 @@ def update
     # create screenshot for manuscript if necessary
     changed_manuscript = @medium.manuscript_data != old_manuscript_data
     if @medium.manuscript.present? && changed_manuscript
+      if @medium.released
+        @medium.file_last_edited = @medium.updated_at
```

Comment:
To make it more ruby-esque one would probably write this as
`@medium.file_last_edited = @medium.updated_at if @medium.released`